src/Pusher.php

```diff
@@ -250,7 +271,7 @@ private function create_curl($domain, $s_url, $request_method = 'GET', $query_pa
 
         $full_url = $domain.$s_url.'?'.$signed_query;
 
-        $this->log('INFO: create_curl( '.$full_url.' )');
+        $this->log('create_curl( {full_url} )', compact('full_url'));
```

Comment:
I know it's more typing, but I think `array('full_url' => $full_url)` is conceptually lot simpler/clearer.